batch/2.0/_index.md

```diff
@@ -1,24 +1,27 @@
 ---
-title: "Jakarta Batch 2.0 (milestone)"
-date: 2020-06-09
+title: "Jakarta Batch 2.0"
+date: 2020-07-25
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Batch specifies a Java API plus an XML-based job specification language (JSL), which lets you compose batch jobs in XML from reusable Java application artifacts and conveniently parameterize different executions of a single job.
 
 * [Jakarta Batch 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.batch/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Batch 2.0 Specification Document](./batch-spec-2.0-draft.pdf) (PDF)
-* [Jakarta Batch 2.0 Specification Document](./batch-spec-2.0-draft.html) (HTML)
+* [Jakarta Batch 2.0 Specification Document](./jakarta-batch-spec-2.0.pdf) (PDF)
+* [Jakarta Batch 2.0 Specification Document](./jakarta-batch-spec-2.0.html) (HTML)
 * [Jakarta Batch 2.0 Javadoc](./apidocs)
-* [Jakarta Batch 2.0 TCK](https://download.eclipse.org/jakartabatch/tck/eftl/jakarta.batch.official.tck-2.0.0-M2.zip) (sig - N/A, [sha](https://download.eclipse.org/jakartabatch/tck/eftl/jakarta.batch.official.tck-2.0.0-M2.zip.sha256), pub - N/A):
+* [Jakarta Batch 2.0 TCK](https://download.eclipse.org/jakartaee/batch/2.0/jakarta.batch.official.tck-2.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/batch/2.0/jakarta.batch.official.tck-2.0.0.zip.sig), [sha](https://download.eclipse.org/jakartaee/batch/2.0/jakarta.batch.official.tck-2.0.0.zip.sha256), [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
Why is there an "official" in the zip name?

Reply:
In the Spec. committee meeting at Aug 12, we discussed about the form of zip file name,
and we agreed that the folder path is required, the file name pattern is preferred.